packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -638,6 +638,55 @@ void main() {
     expect(expandedContainerDecoration.shape, shape);
   });
 
+  testWidgets('ExpansionTile shape from ExpansionTileTheme', (WidgetTester tester) async {
```

Comment:
Is your goal here to test the local `ExpansionTileTheme` for the shape? 

If yes, this can should be the same as other expanded and collapsed tests in the theme class but using a local theme, which also tests other theme values.

Theme test doesn't belong in `expansion_tile_test.dart` but theme class.

Reply:
Ohh yeah.. I will move it to `expansion_tile_test.dart`